Docs/troubleshooting.md

```diff
@@ -0,0 +1,236 @@
+
+
+#Troubleshooting
+Provide some tips and tricks to help customize and troubleshoot the solution.
```

Comment:
I am unsure about the meaning of this file. Having a troubleshooting listing the FAQ is a great addition I think! However what follows in the rest of file look more like a description of the different features we offer with the kit. Wouldn't it make sens to split both of them?

Reply:
yes, I agree see Sascha comment too

Reply:
I am rephrasing this doc to a Cheat Sheet, it will serve as a How To guide that will contain common configuration setting and setup instructions that are not included in the quick start.

Reply:
That sounds good. And please make sure to take the parts you want to move from the quick start to the cheat sheet from the current DEV branch here and not your fork...!